clippy_lints/src/methods/mod.rs

```diff
@@ -197,13 +197,24 @@ declare_clippy_lint! {
     /// result.unwrap();
     /// ```
     ///
-    /// Use instead:
+    /// If [expect_used](#expect_used) is allowed, instead:
     /// ```rust
     /// # let option = Some(1);
     /// # let result: Result<usize, ()> = Ok(1);
     /// option.expect("more helpful message");
     /// result.expect("more helpful message");
     /// ```
+    ///
+    /// Otherwise try using
```

Comment:
The `expect_used` lint is allow-by-default, so it would be better to show the case where this is enabled.
```suggestion
    /// If [expect_used](#expect_used) is enabled, instead:
```